themes/default/layouts/partials/header.html

```diff
@@ -22,77 +22,72 @@
                     </a>
                 </div>
-
                 <ul class="desktop-menu">
                     <li class="header-nav-item-drop-down">
-                        <span class="dropdown-title">Cloud Engineering</span>
+                        <span class="dropdown-title whitespace-nowrap">Cloud Engineering</span>
                         <span class="dropdown-menu-caret"></span>
                         <div class="header-nav-drop-down-menu">
                             <ul>
-                                <a href="{{ relref . "/cloud-engineering" }}">
-                                    <li class="header-dropdown-item mb-3">
+                                <li>
+                                    <a href="{{ relref . "/cloud-engineering" }}">
                                         Overview
-                                    </li>
-                                </a>
-                                <a href="{{ relref . "/solutions" }}">
-                                    <li class="header-dropdown-item my-3">
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="{{ relref . "/solutions" }}">
                                         Solutions
-                                    </li>
-                                </a>
-                                <a href="{{ relref . "/case-studies" }}">
-                                    <li class="header-dropdown-item mt-3">
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="{{ relref . "/case-studies" }}">
                                         Case Studies
-                                    </li>
-                                </a>
+                                    </a>
+                                </li>
                             </ul>
                         </div>
                     </li>
-
                     <li>
                         <a href="{{ relref . "/product" }}">Products</a>
                     </li>
-
                     <li>
                         <a href="{{ relref . "/pricing" }}">Pricing</a>
                     </li>
-
                     <li>
                         <a href="{{ relref . "/docs" }}">Docs</a>
                     </li>
-
                     <li class="header-nav-item-drop-down">
                         <span class="dropdown-title">Learn</span>
                         <span class="dropdown-menu-caret"></span>
-                        <div class="header-nav-drop-down-menu learn-menu">
+                        <div class="header-nav-drop-down-menu">
                             <ul>
-                                <a href="{{ relref . "/blog" }}">
-                                    <li class="header-dropdown-item mb-3">
```

Comment:
Noticed this was invalid (`ul > a > li` rather than `ul > li > a`) so fixed it up.